server/middleware/validationMiddleware.test.ts

```diff
@@ -32,17 +32,6 @@ describe('Validation middleware', () => {
     expect(next).toHaveBeenCalled()
   })
 
-  it.skip('should handle unexpected exceptions', async () => {
-    const alwaysFailsValidator: Validator = async () => {
-      throw new Error('Unexpected error')
-    }
-    await validationMiddleware(alwaysFailsValidator)(req, res, next)
-
-    expect(next).toHaveBeenCalledWith(new Error('Unexpected error'))
-    expect(req.flash).not.toHaveBeenCalled()
-    expect(req.errors).toBeUndefined()
-  })
-
```

Comment:
This was an old test that was being skipped because it couldn't be implemented. So have removed it